ERCS/erc-7634.md

```diff
@@ -0,0 +1,208 @@
+---
+eip: 7634
+title: Limited Transferrable NFT
```

Comment:
This is a good title. If you want to make it a great title, try and cram a bit more detail about the limitation these NFTs have.